src/dapps/nft.ts

```diff
@@ -74,6 +74,8 @@ export type NFTFilters = {
   isWearableSmart?: boolean
   wearableCategory?: WearableCategory
   wearableGenders?: WearableGender[]
+  emoteCategory?: EmoteCategory
+  emoteGenders?: WearableGender[]
```

Comment:
Should we need to rename to baseGenders or have a new type `emoteGender` equal to `wearableGender`?